tutor/src/screens/task/step/instructions.js

```diff
@@ -9,6 +9,12 @@ import Time from '../../../components/time';
 
 const CardBody = styled(InnerStepCard)`
   padding-bottom: 5rem;
+  /* reading assignment has a sticky navigation bar at the bottom */
+  /* all other assignments, make this div to fill up the screen (the header is 20vh) */
+  ${({ theme, taskType }) => taskType !== 'reading' && theme.breakpoint.only.mobile`
+    height: 80vh;
```

Comment:
I might be thinking of the wrong screen here, but you could use `calc` here to subtract 100vh from `navbars.top` + `navbars.readingSubnav` from the theme vars just in case those change in the future.

Reply:
it works! thanks for the advice :)